File: help/pt_pt/exercicios_da_ficha_5/Ficha05_008/Ficha05_008.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha05_008 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Minimo : " );
        int imin = Integer.valueOf(keyb.nextLine());
        System.out.print("Maximo : " );
        int imax = Integer.valueOf(keyb.nextLine());
        for( int i = imin; i < imax + 1; i = i + 1 ) {
            if(i % 2 == 0 ){
                System.out.print(i + " " );
            }
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 16:03

File: help/pt_pt/exercicios_da_ficha_5/Ficha05_008/Ficha05_008.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Minimo : ");
    int imin;
    scanf("%d", &imin);
    printf("Maximo : ");
    int imax;
    scanf("%d", &imax);
    int i = 0 ;
    for( int i = imin; i < imax + 1; i = i + 1 ) {
        if(i % 2 == 0 ){
            printf("%d ",i );
        }
    }
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 16:03